ArduCopter/Copter.h

```diff
@@ -629,7 +629,6 @@ class Copter : public AP_Vehicle {
         DISABLE_THRUST_LOSS_CHECK     = (1<<0),   // 1
         DISABLE_YAW_IMBALANCE_WARNING = (1<<1),   // 2
         RELEASE_GRIPPER_ON_THRUST_LOSS = (1<<2),  // 4
-        REQUIRE_POSITION_FOR_ARMING =   (1<<3),   // 8
```

Comment:
Should we keep the enum value to prevent the bit being used in the future? EG "REQUIRE_POSITION_FOR_ARMING_UNUSED". 

It might be a good idea even if we don't do the conversion, it really depends on how exciting the next option we want to add it.

Reply:
@rmackay9 mentioned this hasn't gone out in a stable release.  It's a reasonable question, 'though.